### PR TITLE
Detail-window-popup

### PR DESCRIPTION
**Pull request by elad237**
updated the style.css file , popup.js file  img folder and I added the popup.css to create the detail window popup

### CODE REVIEW

File: popup.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    const filteredProject = projects.filter((project) => project.id == e.target.id)
+    popUp.innerHTML = displayPopUp(filteredProject[0]);
+    popUp.classList.remove('hidden')
+  }
+});
+
+popUp.addEventListener('click', (e) => {
+  if(e.target.classList.contains('fa')) {
+    popUp.classList.add('hidden')
+    console.log(e.target.id)
```
Comment (mckent05) [inline popup.js]: - To make your project more maintainable, it is not advisable to print anything in the console, I'd suggest you remove all `console.log()` statements. 🙏 🙏

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  projectContainer.innerHTML += displayProjectCard(project);
+})
+
+projects.forEach((project) => {
+  popUp.innerHTML = displayPopUp(project);
+})
+
+projectContainer.addEventListener('click', (e) => {
+  if(e.target.classList.contains('project-item-btn')) {
+    console.log(e.target.id)
```
Comment (mckent05) [inline popup.js]: - To make your project more maintainable, it is not advisable to print anything in the console, I'd suggest you remove all console.log() statements. 🙏 🙏

File: popup.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        See live<img src="img/liveDemo.svg" />
+      </a>
+      <a class="see-project" id="source" href=${object.linkToSource}>
+        See Source <img src="img/gitIcon.png" />
+      </a>
+    </div>
+    <ul class="langues-use" id="technologypopUp">${generateTechnology(object)}</ul>
+    <p class="project-details" id="details">${object.projectDescription}</p>
+  </section>
+  `
```
Comment (mckent05) [inline popup.js]: - Good job on the job done so far 👍 . Your popup still has some little differences from what is in the figma design template. Please reference the image I have attached below 👇 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

# Your project:
-- The close button icon is currently not displaying, please try to fix this issue 😄 
-- Please pay attention to the Project name, it should be aligned with the image (i.e they should have the same margin on the left). Please fix this 👍 .
-- Also, the project buttons on the right should have the same margin on the right side as with the image. Please fix this 🙏 
-- Please ensure that your stick with the general design layout, as regards margins and spacing 👍 👍 
today popup

#Figma Design (Desktop version):
popup figma2



(Mobile version): Also, there are some variations in the mobile version of your page, please reference the image of your project attached below:
-- The project buttons (live and source) aren't aligned as they are in the design template, please fix this 👍 👍 
-- Remember to stick to the design layout with regards to the margins and spacing, please fix this 😄 
today popup mobile

## Figma template Mobile:
popup figma mobile

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @elad237,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing correctly ✔️ ✔️ 
- GitHub flow was correctly used 👏 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @elad237, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights✔

- Linter checks are passing
- Good commit message
- Adherence to GitHub flow
- Requested change has been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**